### PR TITLE
Update di52 to the latest version

### PR DESCRIPTION
**Pull request by borkweb**
This resolves a fatal caused by mismatched di52 versions in WPStaging Pro.

:ticket: [BTRIA-650]
Screenshot: http://p.tri.be/EjS3O1

:point_up: This should do fine for the artifact. If the singleton/bind stuff from di52 wasn't working, the calendar wouldn't render. Automated tests will cover the rest of the requirements for artifacts on such a base-level change.

[BTRIA-650]: https://moderntribe.atlassian.net/browse/BTRIA-650

### CODE REVIEW

**Review by mitogh — APPROVED** [review]
:ok_hand:

**Comment by borkweb** [comment]
Not sure why the dependency checking test is failing. I'll take a peek at that at some point tomorrow.

**Comment by borkweb** [comment]
The good news is that this PR isn't causing the dependency error. The bad news is that I tested this against `master` and it has this same failure:

```
There was 1 failure:

---------
1) Parent_Plugin_RequirementsCest: Should_show_a_notice_if_addon_version_is_lower_than_required_by_parent
 Test  tests/dependency/Parent_Plugin_RequirementsCest.php:should_show_a_notice_if_addon_version_is_lower_than_required_by_parent
 Step  See element ".tribe-notice.tribe-dependency-error[data-plugin="the-events-calendar"]"
 Fail  Element located either by name, CSS or XPath element with '.tribe-notice.tribe-dependency-error[data-plugin="the-events-calendar"]' was not found.

Scenario Steps:

 5. $I->seeElement(".tribe-notice.tribe-dependency-error[data-plugin="the-events-calendar"]") at tests/dependency/Parent_Plugin_RequirementsCest.php:51
 4. $I->amOnPluginsPage() at tests/dependency/Parent_Plugin_RequirementsCest.php:49
 3. $I->loginAsAdmin() at tests/dependency/Parent_Plugin_RequirementsCest.php:48
 2. $I->set_active_plugins(["test-23d4718932b170161ab1dbbc647d50fe/test-23d4718932b170161ab1dbbc647d50fe.php","the-events-calendar/the-events-calendar.php","dependency-...]) at tests/dependency/Parent_Plugin_RequirementsCest.php:46
 1. $I->have_plugin_with_template_and_data("main_and_addon_filter",{"parent_class":"Tribe__Events__Main","addon_class":"DT_Plugin","parent_requires":"4.8","addon_versio...}) at tests/dependency/Parent_Plugin_RequirementsCest.php:43


FAILURES!
Tests: 2, Assertions: 2, Failures: 1.
```

**Review by Camwyn — APPROVED** [review]
🚀 
Looks like @sc0ttkclark had the 🧙

**Review by sc0ttkclark — DISMISSED** [review]
All good unless we need to address the currently existing dependency test failure

File: readme.txt [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 == Changelog ==
 
+= [TBD] TBD =
+
+* Tweak - Update di52 containers to latest version for compatibility with WPStaging Pro [BTRIA-650]
```
Comment (sc0ttkclark) [inline readme.txt:7]:
```suggestion
* Tweak - Update di52 containers to latest version for compatibility with WPStaging Pro. [BTRIA-650]
```

**Pull request merged**